rewrite-java/src/main/java/org/openrewrite/java/AddOrUpdateAnnotationAttribute.java

```diff
@@ -82,7 +82,7 @@ public J.Annotation visitAnnotation(J.Annotation a, ExecutionContext ctx) {
 
                 String newAttributeValue = maybeQuoteStringArgument(attributeName, attributeValue, a);
                 List<Expression> currentArgs = a.getArguments();
-                if (currentArgs == null || currentArgs.isEmpty()) {
+                if (currentArgs == null || currentArgs.isEmpty() || currentArgs.get(0) instanceof J.Empty) {
```

Comment:
Should this perhaps be changed in the parser instead of here?
I'd assume when you use `@Foo` currentArgs = null and when you use `@Foo()` currentArgs = empty

Reply:
We use that J.Empty when we want to be explicit about the presence of those parentheses; I think this change is good to merge. Thanks both!

Reply:
Thanks for review!